addons/actions/src/index.js

```diff
@@ -1,6 +1,8 @@
+import { action, configureActions, decorateAction } from './preview';
+
 // addons, panels and events get unique names using a prefix
 export const ADDON_ID = 'storybook/actions';
 export const PANEL_ID = `${ADDON_ID}/actions-panel`;
 export const EVENT_ID = `${ADDON_ID}/action-event`;
 
-export * from './preview';
```

Comment:
Yeah, unfortunately this `export *` syntax doesn't play well with `transform-runtime` https://github.com/babel/babel/issues/2877